package.json

```diff
@@ -62,6 +62,7 @@
     }
   ],
   "dependencies": {
+    "babel-polyfill": "^6.23.0",
```

Comment:
Even when we use babel-register the poly is needed so we can move it to devDependencys :+1: 

>Polyfill not included
You must include the polyfill separately when using features that require it, like generators.